### DISCUSSION TITLE
What is the Expected output?

### DISCUSSION THREAD

**Discussion by mazrrkhan** [provinzio/CoinTaxman#123] [Q&A] [answered]
Hey guys,

thanks for sharing this, wondering what does the output look like? I couldnt find anything in the documentation

**Comment by provinzio** [comment, marked as answer]
At the moment: the terminal prints a summarized tax evaluation and a csv file with tax relevant information is saved in `\export\`.

After an evaluation `make archive` creates a zip file in the export-dir with all relevant information regarding the evaluation.

**Reply by mazrrkhan** [reply]
Thanks
